pom.xml

```diff
@@ -29,7 +30,7 @@
     </dependency>
   </dependencies>
 
-  <scm>
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
```

Comment:
For the record, this is only strictly necessary in Maven multi-module projects, although it is harmless to have it in a single-module project as well (as it is in the archetype).